src/client/sandbox/event/simulator.js

```diff
@@ -531,8 +531,8 @@ export default class EventSimulator {
 
             // NOTE: In IE11, iframe's window.event object is null. We need to set
             // iframe's window.event object manually by using window.event (B254199).
-            if (browserUtils.version === 11 && isElementInIframe) {
-                Object.defineProperty(domUtils.getIframeByElement(el).contentWindow, 'event', {
+            if (browserUtils.version === 11 && isElementInIframe && iframe) {
```

Comment:
`&& isElementInIframe` check is excess.
See https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/event/simulator.js#L440